src/partition.rs

```diff
@@ -709,7 +709,7 @@ ref_frame: usize, mv: &MotionVector, bit_depth: usize) {
                 sum += intermediate[r + k][c] as i32 * SUBPEL_FILTERS[y_filter_idx][row_frac as usize][k];
               }
               let output_index = r * stride + c;
-              let val = ((sum + 8) >> 4).max(0).min(max_sample_val);
+              let val = ((sum + 1024) >> 11).max(0).min(max_sample_val);
```

Comment:
So, its not shift right by 7 + 7 = 14 bits?

Reply:
Oh, the 1st stage already did 3 bits above. So, 3 + 11 =14 bits.

Reply:
If we use higher precision vars for sum and intermediate, can we even shift less at 1st stage?

Reply:
the spec tells you how much to shift. No choice here...